addon/object/index.js

```diff
@@ -179,7 +191,17 @@ export const computed = decoratorWithParams(function(target, key, desc, params)
 
     // Use a standard ember computed since getter/setter arrity is restricted,
     // meaning ember-macro-helpers doesn't provide any benefit
-    return emberComputed(...params, { get, set: setter });
+    if (SUPPORTS_NEW_COMPUTED) {
+      return emberComputed(...params, { get, set: setter });
+    } else {
+      return emberComputed(...params, function() {
```

Comment:
Im thinking of this code: https://github.com/emberjs/ember.js/blob/v1.11.0/packages/ember-metal/lib/computed.js#L121-L123